kinetic/distribution.yaml

```diff
@@ -2432,6 +2432,24 @@ repositories:
       url: https://github.com/OctoMap/octomap.git
       version: devel
     status: developed
+  octomap_mapping:
+    doc:
+      type: git
+      url: https://github.com/OctoMap/octomap_mapping.git
+      version: jade-devel
+    release:
+      packages:
+      - octomap_mapping
+      - octomap_server
+      tags:
+        release: release/jade/{package}/{version}
```

Comment:
You can't just copy the entries from Jade. These repos need to be released for Kinetic using `bloom`.